lib/adhearsion/configuration.rb

```diff
@@ -53,6 +54,10 @@ def initialize(&block)
           formatter nil, :desc => <<-__
             A log formatter to apply to all active outputters. If nil, the Adhearsion default formatter will be used.
           __
+          default_formatter ::Logging::Layouts::Pattern, :desc => <<-__
+            A log layout to apply to stdout outputters.
```

Comment:
2 qs here.  Can we name this simply "layout" instead of default_formatter?

Also, can you apply this to `lib/adhearsion/initializer.rb` as well?  Then the above comment can read that it applies to _all_ outputters.

Reply:
maybe `formatter_class` to keep with adhearsion's terminology (adhearsion decided to alias `format` and `layout`)? here we have their `formatter` (an optionally instantiated Logging::Layout), and `our_thing` (a Logging::Layout class of our choosing, for later instantiation). What do you think?

Reply:
i'd update the comment to match the jargon as well. i.e., "A log formatter class to instantiate for all active outputters."

Reply:
Oh dear! I didn't realize that the existing formatter option pertained to layouts.  Can we just use that?

Sorry for potentially created unnecessary work here, I simply didn't realize that formatter == layout.  It looks like this is an artifact of backwards-compatibility, following the port from Log4r to logging adhearsion/adhearsion@91a805290

Reply:
@sfgeorge, so we can't _just_ use `formatter`, without doing something like the earlier strategy from earlier in this experiment.  i.e., pass in an instantiated Layout with color-scheme that gets applied to all outputters, and subsequently remove the color-scheme from the outputter used for stdout.

Reply:
... if you're getting dizzy, then it means you're paying attention 🍝